igvc_platform/launch/imu.launch

```diff
@@ -1,21 +1,9 @@
 <!-- imu.launch -->
 <launch>
-    <node name="imu" pkg="igvc_platform" type="imu" output="screen" >
-      <param name="yaw_offset" type="double" value="0"/>
-      <!-- old yaw values -->
-      <!-- 1 x 2.85023247974 -->
-      <!-- 1x -2.8440947155 -->
-      <!-- 2.7091530674 -->
-      <!-- 3.11713869703 -->
-      <!-- 2.87548612285 -->
-      <!-- 2.7670798078 -->
-      <!-- 2.93174942492 -->
-      <!-- 2.69890706297 -->
-      <!-- 3.09938308897 -->
-      <!-- 2.93163273405 -->
-      <!-- 2.73621983653 -->
-      <!-- at comp -->
-      <!-- 1.0726804111 -->
-      <!-- 1.36263847313 -->
-    </node>
+    <node name="imu" pkg="igvc_platform" type="imu" output="screen" />
+    	<!-- Matrix to put IMU in same coordinate system as the robot. -->
+	    <rosparam param="imu_orientation_correction">[1, 0, 0, 0, -1, 0, 0, 0, -1]</rosparam>
```

Comment:
I would line this up as a matrix, just for quicker scanning

Reply:
Done